ci/pipelines/build-and-deploy.yml

```diff
@@ -158,9 +158,10 @@ jobs:
               cp ../whitelisted-emails/domains.yml data/domains.yml
               cp ../organisations-list/organisations.yml data/organisations.yml
               bundle exec middleman build
-              cp -r . ../bundled/
+              cp -r build ../staticsite
```

Comment:
Since the static files are still under `build/` and you are removing the path from the manifest.yml, will it still know where the files are located?

Reply:
The content of build is now copied into staticsite rather than having `staticsite/build`